Annotations/Uri Annotation/UriAnnotation/MainWindow.xaml.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Syncfusion.Pdf.Graphics;
+using Syncfusion.Pdf.Interactive;
+using Syncfusion.Pdf.Parsing;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace UriAnnotation
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        public RectangleF annotationBounds = RectangleF.Empty;
+        public int annotationPageIndex = -1;
+        ToolTip tooltip = new ToolTip();
+        PdfUnitConvertor convertor = new PdfUnitConvertor();
+        public MainWindow()
+        {
+            InitializeComponent();
+            docView.Load("../../../../../PDF/HTTP Succinctly.pdf");
+        }
+        private void BtnLoad_Click(object sender, RoutedEventArgs e)
+        {
+            PdfLoadedDocument lDoc = docView.LoadedDocument;
+
+            //Set the standard font.
+            PdfFont font = new PdfStandardFont(PdfFontFamily.Helvetica, 20);
+
+            //Draw the border.
+            docView.LoadedDocument.Pages[0].Graphics.DrawRectangle(PdfPens.Red, new RectangleF(10, 10, 30, 30));
+
+
+            //Create a new Uri Annotation.
+            PdfUriAnnotation uriAnnotation = new PdfUriAnnotation(new RectangleF(10, 10, 30, 30), "http://www.google.com");
+            //Set Text to uriAnnotation.
+            uriAnnotation.Text = "Uri Annotation";
+            //Add this annotation to a new page.
+            docView.LoadedDocument.Pages[0].Annotations.Add(uriAnnotation);
+            //Save the document to disk.
+            MemoryStream stream = new MemoryStream();
+            lDoc.Save(stream);
+            docView.Load(stream);
+            annotationPageIndex = 0;
+            annotationBounds = uriAnnotation.Bounds;
+            //annotationBounds = convertor.ConvertToPixels(annotationBounds, PdfGraphicsUnit.Point);
```

Comment:
* Remove the commented lines
* Remove unwanted namespaces
* Add comments wherever possible as it is a public repo

Reply:
I have addressed the provided feedbacks.